src/lib.rs

```diff
@@ -21,7 +21,13 @@ pub use type_map;
 
 use runtime::{JsRuntime, JsRuntimeApi};
 
-pub struct JsScriptingPlugin;
+#[derive(Default)]
+pub struct JsScriptingPlugin {
+    /// By default, the plugin will setup script functions corresponding to each [`CoreStage`] to
+    /// run at the start of each stage. This disables that behavior so that script stages must be
+    /// added manually using [`run_script_fn_system`].
+    pub skip_core_stage_setup: bool,
```

Comment:
In my scenario I disabled it so that I could bind `update` to my game's `FixedUpdate` stage, because I actually didn't want scripts to be able to execute outside of fixed update states.